docs/se/observability.adoc

```diff
@@ -0,0 +1,318 @@
+
+|`/health/check/{name}`
+|`GET`
+|Returns all checks for service `name`
+
+|`/health/ready`
+|`HEAD`
+|Returns Service Readiness without details
+
+|`/health/live`
+|`HEAD`
+|Returns whether the service is alive without details
+
+|`/health/started`
+|`HEAD`
+|Returns whether the service is started without details
+
+|`/health/ready/{name}`
+|`HEAD`
+|Returns Service `name` Readiness without details
+
+|`/health/live/{name}`
+|`HEAD`
+|Returns whether the service `name` is alive without details
+
+|`/health/started/{name}`
+|`HEAD`
+|Returns whether the service `name` is started without details
+
+|`/health/check/{name}`
+|`HEAD`
+|Returns all checks for service `name` without details
+
+|===
+
+For more information, please, check xref:{rootdir}/se/health.adoc[Health] documentation.
+
+
+==== Information Observability
+
+Info observability allows configuration of custom properties to be available to users.
+Information observability defines the following endpoints:
+
+|===
+|Endpoint |Method |Action
+
+|`/info`
+|`GET`
+|Returns the Application information
+
+|`/info/{name}`
+|`GET`
+|Returns the Application information for the specified `name`
+
+|===
+
+
+==== Logger Observability
+
+Log observability allows reading and configuring of log levels of various loggers and reading log messages.
+Logger Observability defines the following endpoints:
+
+|===
+|Endpoint |Method |Action
+
+|`/log`
+|`GET`
+|Stream logs (if enabled)
+
+|`/log/loggers`
+|`GET`
+|Returns all logger handlers
+
+|`/log/log/loggers/{logger}`
+|`GET`
+|Returns the Logger by name `logger`
+
+|`/log/loggers/{logger}`
+|`POST`
+|Set Logger level by name `logger`
+
+|`/log/loggers/{logger}`
+|`DELETE`
+|Unset the specified logger `logger`
+
+|===
+
+==== Metrics Observability
+
+Helidon distinguishes among three general _types_, or scopes, of metrics.
+
+.Types (scopes) of metrics
+[%autowidth]
+|====
+| Type/scope | Typical Usage
+
+| base | OS or Java runtime measurements (available heap, disk space, etc.).
+| vendor | Implemented by vendors, including the `REST.request` metrics and other key performance indicator measurements.
+| application | Declared via annotations or programmatically registered by your service code.
+|====
+
+When you add the metrics dependency to your project, Helidon automatically provides a built-in REST endpoint `/observe/metrics` which responds with a report of the registered metrics and their values.
+
+Clients can request a particular output format.
+
+.Formats for `/observe/metrics` output
+[%autowidth]
+|====
+| Format | Requested by
+
+| OpenMetrics (Prometheus) | default (`text/plain`)
+| JSON | Header `Accept: application/json`
+|====
+
+Clients can also limit the report by appending the metric type to the path:
+
+* `/observe/metrics/base`
+* `/observe/metrics/vendor`
+* `/observe/metrics/application`
+
+For more information, please, check xref:{rootdir}/se/metrics/metrics.adoc[Metrics] documentation.
```

Comment:
[standard] For more information see <link>.

Reply:
fixed